### PR TITLE
Add dashboard and early version of match query

### PR DESCRIPTION
**Pull request by julianvogels**

### CODE REVIEW

**Review by HyugaRED — APPROVED** [review]

File: views/dashboard.pug [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+section.jumbotron.d-flex.flex-row.justify-content-end
+    h1.mr-5.text-info 
+      | Dashboard 
+section.container 
+  div.remark
+    p User logged in with email "#{user.get("email")}".
+  form.column(method='post' action='/eventBuddyRequest')
+    .form-group.col-md-5
+      label.mr-sm-2(for='inlineFormCustomSelect') Preference
+      select.custom-select.mr-sm-2(id='event_type' name='event_type') Event type
```
Comment (nachtvlinder) [inline views/dashboard.pug:12]: These are the event-types that were on the buddy sign up form jule shared friday evening: ["Theater_Kunst", "Ausflüge_Cafe", "Sport_Action", "Konzerte_Festivals", "Natur_Sightseeing", "Andere"]
This looks just as good, although splitting nature and sightseeing might make it harder to import the data from the 3700 volunteers, so I'd keep them together for now. And we should keep in mind that the labels are clear (e.g. sports is both doing and watching, café is also excursions, concert is also festival (not everyone will makethat link)).
And adding something for Theatre and art and for 'other' so the existing data can be applied upon.

Comment (nachtvlinder) [reply views/dashboard.pug:12]: @julianvogels

File: views/eventForm.pug [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-        option(value='1') One
-        option(value='2') Two
-        option(value='3') Three
     .col-7
       input.form-control(type='string' placeholder='Title')
     .col
       input.form-control(type='Date' placeholder='Date')
     .col
       input.form-control(type='string' placeholder='Description')
+    .col-md-5
```
Comment (nachtvlinder) [inline views/eventForm.pug]: - [ ] adjust event types (see comment on dashboard.pug)

File: views/signupVolunteer.pug [hunk 3]
```diff
@@ -42,6 +42,20 @@ form.column(method='post' action='')
       option(value="intellectual_disability_awereness") Intellectual disability awereness
       option(value="speech_with_gestures") Speech with gestures
       option(value="LGBTQIAplus_awareness") LGBTQIAplus awareness
+  .form-group.col-md-5
+    label(for='event_types') I am interested in these event types
+    select.form-control.col-md-5(id='event_types' name='event_types' multiple)
```
Comment (nachtvlinder) [inline views/signupVolunteer.pug:68]: - [x] adjust event type (see comment on dashboard.pug)

File: views/dashboard.pug (hunk 1 above)
Comment (julianvogels) [reply views/dashboard.pug:12]: Fixed!

**Review by HyugaRED — APPROVED** [review]

**Pull request merged**